Shared/Features/Freaks/View/FreaksView.swift

```diff
@@ -7,16 +7,35 @@ struct FreaksView: View {
         let coloredNavAppearance = UINavigationBarAppearance()
         coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor(Color("SecondaryColor"))]
         coloredNavAppearance.backgroundColor = UIColor(Color("AccentColor"))
-        
+
         UINavigationBar.appearance().standardAppearance = coloredNavAppearance
         UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
     }
 
     var body: some View {
         NavigationView {
-            List(viewModel.freaks, id: \.firstName) { freak in
-                NavigationLink(destination: FreakDetailsView(viewModel: FreakDetailsViewModel(freak: freak))) {
-                    ListItemView(viewModel: ListItemViewModel(freak: freak))
+            VStack(spacing: 0) {
+                HStack {
+                    Spacer()
+                    Button("Skills") {
+                        print("skills")
+                    }
+                    Spacer()
```

Comment:
Also, can we have some kind of divider between the Skills button and the Projects button?